src/triage/component/postmodeling/contrast/evaluator.py

```diff
@@ -0,0 +1,543 @@
+"""
+Model Evaluator
```

Comment:
This is not the name of the class in this file!

Also the name of the file is deceiving